packages/fiber/demo/app.json

```diff
@@ -0,0 +1,22 @@
+{
+  "expo": {
+    "name": "react-three-fiber-demo",
+    "slug": "react-three-fiber-demo",
+    "privacy": "public",
+    "sdkVersion": "41.0.0",
+    "entryPoint": "demo/AppEntry",
```

Comment:
```suggestion
```

We recommend against using this as it doesn't work with other RN tools.

Reply:
Hi. I've tried removing this line, but then I find it pretty impossible to start the demo project within the larger directory.
I tried creating a package.json with this following content
````json
{
  "name": "r3fexpodemo",
  "version": "0.0.1",
  "private": true,
  "main": "./AppEntry.tsx",
  "scripts": {
    "start": "export EXPO_NO_TYPESCRIPT_SETUP=1 && expo start"
  },
  "dependencies": {
    "expo": "^41.0.0",
    "react": "16.13.1",
    "react-dom": "16.13.1",
    "react-native": "https://github.com/expo/react-native/archive/sdk-41.0.0.tar.gz"
  },
  "devDependencies": {
    "@types/react": "~16.9.35",
    "@types/react-native": "~0.63.2"
  }
}
````
tried `npm install` and such, but since the demo project is importing from directories directly outside of it (there's a `import { Canvas, useFrame } from '../src/native'`) in the demo, building fails with the error of `Unable to resolve module @babel/runtime/helpers/interopRequireWildcard from /<projectPath>/react-three-fiber/packages/fiber/src/native.tsx:`
Since it's just a demo project could we keep this as-is? If not could you provide an alternative?